packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form.feature

```diff
@@ -0,0 +1,93 @@
+    Then <participantType> radio buttons appear
+    When they select a <participantType>
+    Then an <autoSelectedTaxForm> appears in the Docusign iframe
+
+    Examples: 
+      | participantType       | autoSelectedTaxForm |
+      | individualParticipant | W8-BEN              |
+      | businessEntity        | W8-BEN-E            |
+
+  @minutia @ui
+  Scenario Outline: Text copies change based on tax form participant fills out
+    When they are required to fill out <typeTaxForm>
+    Then the <taxFormSubHeader> displays with <typeTaxForm>
+    And the <taxFormDescriptionCopy> / <changeTaxFormCopy> changes depending on the <typeTaxForm>
+
+    Examples: 
+      | typeTaxForm | taxFormSubHeader  | taxFormDescriptionCopy                                                                                                   | changeTaxFormCopy                                                |
+      | W9          | W9 Tax Form       | Participants based in the US and partnering with US-based brands need to submit a W9 form.                               | Not based in the US?                                             |
+      | W8-BEN      | W8-BEN Tax Form   | Individuals residing outside of the US, joining the referral program of a US-based cmpany, need to submit a W8-BEN form. | Represent a business entity or you're based in the US?           |
+      | W8-BEN-E    | W8-BEN-E Tax Form | Participants residing outside of the US who represent a business enttiy need to submit a W8-BEN-E form.                  | Joining this program as an individual or you're based in the US? |
+
+  @minutia
+  Scenario: Participant successfully submits Docusign document and is directed to summary page
+    When they successfully fillout and submit the Docusign document within the iframe
+    And they press "Continue" or refresh the page
+    Then they will be directed to the Document Submitted page
+
+  @minutia @ui
+  Scenario: Docusign iframe is loading
+    When the Docusign iframe is loading the Tax Form
+    Then the iframe displays a spinnder inside the igrame
+    When the Docusign iframe loads in the Tax Form
+    Then the loader dissapers and the participant can see the Tax Form
+
+  @minutia
+  Scenario: Participant completes Docusign form
+    When they fillout and complete the Docusign form inside the iframe
+    Then the Docusign iframe session completes with one of the following success statuses:
+      | signing_complete |
+      | viewing_complete |
+    Then they are redirected to step 4
+
+  @minutia @ui
+  Scenario: Participant's Docusign session expires
+    When they have the Docusign form open in the iframe
+    And they do not interact with the iframe for more than 20 minutes
+    Then the Docusign iframe session expires with one of the following expirey statuses:
+      | session_timeout |
+      | ttl_expired     |
+    And the Tax Form dissapers
+    Then they see a Docusign expired session notification inside the iframe
```

Comment:
Is this inside the iframe or is the iframe hidden and this displayed? Small detail